src/components/login.tsx

```diff
@@ -52,56 +51,91 @@ export class LoginScreenComponent extends React.Component<LoginProps, LoginState
-          />
-
-          <Text>
-            Stay signed in
-          </Text>
-        </View>
-
-        <Text>
-          { this.props.error }
-        </Text>
-
-        <Button title='Login' onPress={this.submit.bind(this)} />
-
-        <Text style={Styles.paragraph} onPress={() => navigate('SignUp')}>
-          Don't have an account? <Text style={Styles.link} >Sign up</Text>
-        </Text>
-      </KeyboardAvoidingView>
+      <Container style={Styles.viewContainer}>
+        <Content>
+          <ImageBackground style={Styles.loginHeader} source={require('../../images/loginHeader.png')}>
+            <Image source={require('../../images/g-logo.png')} style={Styles.loginHeaderImage} />
+
+            <Text style={Styles.loginHeaderTitle}>
+              GENERIC SYSTEMS
+            </Text>
+          </ImageBackground>
+
+          <View style={Styles.loginBody}>
+            <Form style={Styles.form}>
+              <Item style={Styles.formItem} stackedLabel>
+                <Label>{ username ? 'Email' : '' }</Label>
+
+                <Input
+                  style={Styles.formInput}
+                  placeholder='Email'
+                  keyboardType='email-address'
+                  autoCapitalize='none'
+                  autoCorrect={false}
+                  onChangeText={(text) => this.setState({ username: text })}
+                />
+              </Item>
+
+              <Item style={Styles.formItem} stackedLabel>
+                <Label>{ password ? 'Password' : '' }</Label>
+
+                <Input
+                  style={[Styles.formInput, Styles.formInputPass]}
+                  placeholder='Password'
+                  secureTextEntry={true}
+                  onChangeText={(text) => this.setState({ password: text })}
+                />
+
+                <Text style={Styles.formInputLink}>
+                  FORGOT?
+                </Text>
+              </Item>
+
+              <View style={Styles.switchContainer}>
+                <Switch
+                  value={this.state.rememberMe}
+                  onValueChange={(value) => this.setState({rememberMe: value})}
+                />
+
+                <Text style={Styles.switchText}>
+                  Stay signed in
+                </Text>
+              </View>
+            </Form>
+
+            <Text>
+              { this.props.error }
+            </Text>
+
+            <Button style={Styles.formButton} rounded dark onPress={() => this.submit()}>
+              <Text>sign in</Text>
+            </Button>
+
+            <Text style={Styles.formParagraph}>
+              Don't have an account? <Text style={Styles.link} onPress={() => navigate('SignUp')}>Sign up</Text>
+            </Text>
+
+            <Button style={Styles.loginFacebookButton} rounded iconLeft>
+              <Icon name='facebook-f' style={Styles.loginSocialIcon}/>
+
+              <Text uppercase={false}>
+                Log in with Facebook
```

Comment:
Even though they are not functional yet i think it would be good if we moved the facebook and google log in buttons to a seperate component called oauth-signin-buttons or something